### PR TITLE
Add Vultr CLI tool

### PR DESCRIPTION
**Pull request by pcgeek86**
- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by rashil2000** [comment]
Missing:
- checkver
- autoupdate
- autoupdate hash

File: bucket/vultr.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "license": "Apache-2.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/vultr/vultr-cli/releases/download/v2.12.1/vultr-cli_2.12.1_windows_64-bit.zip",
+            "hash": "688fa14d2e057379d90c49749bacc0faa967ec618a31f4448f1ed075537c83e4"
+        }
+    },
+    "bin": [
+        "vultr-cli.exe"
+    ]
```
Comment (tech189) [inline bucket/vultr.json:14]:
```suggestion
    "bin": "vultr-cli.exe",
    "checkver": "github",
    "autoupdate": {
        "architecture": {
            "64bit": {
                "url": "https://github.com/vultr/vultr-cli/releases/download/v$version/vultr-cli_$version_windows_64-bit.zip",
                "hash": {
                    "url": "https://github.com/vultr/vultr-cli/releases/download/v$version/vultr-cli_v$version_checksums.txt"
                }
            }
        }
    }
```

**Review by tech189 — COMMENTED** [review]
Added checkver and autoupdate!

**Pull request merged**